src/gui/widgets/multi_page.hpp

```diff
@@ -69,8 +70,8 @@ class multi_page : public container_base
 	 *                            members. Having both empty and non-empty
 	 *                            id's gives undefined behavior.
 	 */
-	void
-	add_page(const std::map<std::string /* widget id */, string_map>& data);
+	void add_page(const std::map<std::string /* widget id */, string_map>& data);
+	void add_page(const std::string& type, int insert_pos, const std::map<std::string /* widget id */, string_map>& data);
```

Comment:
Needs documentation. (A single doc comment cannot apply to two different functions.)